wrench/src/main.rs

```diff
@@ -363,6 +363,34 @@ fn create_notifier() -> (Box<RenderNotifier>, Receiver<NotifierEvent>) {
     (Box::new(Notifier { tx: tx }), rx)
 }
 
+fn rawtest(mut wrench: Wrench, window: &mut WindowWrapper, rx: Receiver<NotifierEvent>) {
+    {
+        let harness = RawtestHarness::new(&mut wrench, window, &rx);
```

Comment:
nit: any reason to not just do `RawtestHarness::new(...).run()` without an explicit scope?

Reply:
No reason. I just moved this code without modifying it. This should be fixed now though.